src/models/emailConfirmationTokens.js

```diff
@@ -31,21 +34,38 @@ const initialize = (models) => {
     },
   );
 
-  models.email_confirmation_tokens.prototype.confirmEmail = async function () {
+  models.email_confirmation_tokens.prototype.cancelToken = async function () {
```

Comment:
`cancelToken` ismi akla ilk önce "Bu token'ı iptal et" anlamını getirir. Bu işlemde tüm token'ları iptal ettiğimiz için `cancelOtherTokens` daha doğru bir isimlendirme olacaktır.